### PR TITLE
Update time-settings.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to access Timezone in m365 defender portal

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7352

File: microsoft-365/security/defender-endpoint/time-settings.md [hunk 1]
```diff
@@ -37,9 +37,7 @@ Cyberforensic investigations often rely on time stamps to piece together the seq
 
 Microsoft Defender for Endpoint can display either Coordinated Universal Time (UTC) or local time.
 
-Your current time zone setting is shown in the Microsoft Defender for Endpoint menu. You can change the displayed time zone in the **Time zone** menu.
-
-![Time zone settings icon2.](images/atp-time-zone-menu.png).
+Your current time zone setting is shown in the Microsoft Defender Settings. You can change the displayed time zone in the **Time zone** menu Under **Settings > Security center**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/time-settings.md]:
```suggestion
Your current time zone setting is shown in the Microsoft Defender settings. You can change the displayed time zone in the **Time zone** menu Under **Settings > Security center**.
```

File: microsoft-365/security/defender-endpoint/time-settings.md [hunk 2]
```diff
@@ -61,9 +59,9 @@ The Microsoft Defender for Endpoint time zone is set by default to UTC. Setting
 
 To set the time zone:
 
-1. Click the **Time zone** menu ![Time zone settings icon3.](images/atp-time-zone.png).
-2. Select the **Timezone UTC** indicator.
-3. Select **Timezone UTC** or your local time zone, for example -7:00.
+1. Click the **Settings* menu in MIcrosoft 365 Defender Portal[https://security.microsoft.com/] ![Time zone settings icon3.](images/atp-time-zone.png).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/time-settings.md]:
```suggestion
1. Click the **Settings** menu in the [MIcrosoft 365 Defender Portal](https://security.microsoft.com/) ![Time zone settings icon3.](images/atp-time-zone.png).
```

File: microsoft-365/security/defender-endpoint/time-settings.md [hunk 3]
```diff
@@ -61,9 +59,9 @@ The Microsoft Defender for Endpoint time zone is set by default to UTC. Setting
...
+2. Select the **Security center**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/time-settings.md]:
```suggestion
2. Select **Security center**.
```

File: microsoft-365/security/defender-endpoint/time-settings.md [hunk 4]
```diff
@@ -61,9 +59,9 @@ The Microsoft Defender for Endpoint time zone is set by default to UTC. Setting
...
+3. Select **Timezone** and Set the Time zone to either UTC or your local time zone.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/time-settings.md]:
```suggestion
3. Select **Timezone** and set the time zone to either UTC or your local time zone.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**